common/protob/messages-mintlayer.proto

```diff
@@ -0,0 +1,483 @@
+ * @embed
+ */
+message MintlayerAccountTxInput {
+    repeated MintlayerAddressPath addresses = 1;                                // 0 or more addresses for which the corresponding private keys will be used to sign the transaction
+    required uint64 nonce = 2;                                                  // incremental account nonce
+    optional MintlayerAccountSpendingDelegationBalance delegation_balance = 3;  // spend from a delegation balance
+}
+
+/** Data type for account spending from a Delegation Balance
+ * @embed
+ */
+message MintlayerAccountSpendingDelegationBalance {
+    required string delegation_id = 1;             // the delegation id of the account in bech32 encoding
+    required bytes amount = 2;                     // amount of coins to spend
+}
+
+/** Data type for transaction Account Command input to be signed.
+ *
+ * @embed
+ */
+message MintlayerAccountCommandTxInput {
+    repeated MintlayerAddressPath addresses = 1;                             // 0 or more addresses for which the corresponding private keys will be used to sign the transaction
+    required uint64 nonce = 2;                                               // incremental account nonce
+    optional MintlayerMintTokens mint = 3;                                   // mint tokens
+    optional MintlayerUnmintTokens unmint = 4;                               // unmint tokens
+    optional MintlayerLockTokenSupply lock_token_supply = 5;                 // lock token supply
+    optional MintlayerFreezeToken freeze_token = 6;                          // freeze token
+    optional MintlayerUnfreezeToken unfreeze_token = 7;                      // unfreeze token
+    optional MintlayerChangeTokenAuthority change_token_authority = 8;       // change token authority
+    optional MintlayerConcludeOrder conclude_order = 9;                      // conclude an order
+    optional MintlayerFillOrder fill_order = 10;                             // fill an order
+    optional MintlayerChangeTokenMetadataUri change_token_metadata_uri = 11; // change token metadata uri
+}
+
+/** Data type for account command Mint tokens
+ * @embed
+ */
+message MintlayerMintTokens {
+    required string token_id = 1;                  // token id in bech32 encoding
+    required bytes amount = 2;                     // amount of tokens to mint
+}
+
+/** Data type for account command Unmint tokens
+ * @embed
+ */
+message MintlayerUnmintTokens {
+    required string token_id = 1;                  // token id in bech32 encoding
+}
+
+/** Data type for account command Lock token supply
+ * @embed
+ */
+message MintlayerLockTokenSupply {
+    required string token_id = 1;                  // token id in bech32 encoding
+}
+
+/** Data type for account command Freeze token
+ * @embed
+ */
+message MintlayerFreezeToken {
+    required string token_id = 1;                  // token id in bech32 encoding
+    required bool is_token_unfreezeable = 2;        // is unfreezeable after freezing it
```

Comment:
Actually I meant that "unfreezeable" in the comment was a typo. And "unfreezable" in the name was correct.

P.S. There are quite a few places with "unfreezeable" now, but I guess an auto-replace should fix them all.